frontend/public/style/_common.scss

```diff
@@ -581,7 +581,7 @@
   margin: 0 !important;
 }
 
-.text-filter {
+.co-m-pane__filter-bar-group .text-filter {
```

Comment:
I recommend keeping this as is and removing the class when it's not needed.

Reply:
removed